doc/releases/release-notes-2.7.rst

```diff
@@ -117,6 +117,12 @@ Stable API changes in this release
     structure contains two members: ``handles``, which was moved from
     :c:struct:`bt_gatt_read_params`, and ``variable``.
 
+* Networking
```

Comment:
since this has been converted from a bug to an enhancement, does that mean it can no longer go in for v2.7.0 @nashif ?